### ISSUE TITLE
Feature request: Allow me to see the full list of excluded websites

### ISSUE THREAD

**Issue by desbest** [Mr-VincentW/PhotoShow#62]
As the title says

**Comment by Mr-VincentW** [comment]
Thanks for your feedback. This likely will be addressed in PhotoShow's future version.

**Comment by desbest** [comment]
Is there an update on this?

**Comment by Mr-VincentW** [comment]
This issue remains unresolved.
PhotoShow does not have a dedicated options page at the moment, and the popup is too small to accommodate a complete blacklist/whitelist. A fully functional options page requires significant effort, hence this task has been deprioritized in favor of other ongoing, higher-priority tasks.

I'll update this issue once it's addressed. Thanks.

**Comment by garyking** [comment]
Yes, best thing to do would be to create a standalone options page for this extension, because I zoom my Chrome by default to 150%, so all the text on the tiny frame used for this extension's options page is so big that I have to scroll horizontally to see it all.

**Comment by Mr-VincentW** [comment]
Hi, just a quick update — starting from version **4.68.0**, PhotoShow now includes a dedicated **options page** where you'll find all the **global settings** that were previously available in the popup. The full blocklist/whitelist have also been added:

![Image](https://github.com/user-attachments/assets/a1a61b44-604c-4e0e-9bec-58e56b415e59)

The popup now focuses on **site-specific settings**, giving you more control and flexibility to fine-tune behaviors on a per-site basis.

Enjoy!
